tensorflow/contrib/cmake/tf_cc_ops.cmake

```diff
@@ -120,3 +120,41 @@ list(REMOVE_ITEM tf_cc_srcs ${tf_cc_test_srcs})
 
 add_library(tf_cc OBJECT ${tf_cc_srcs})
 add_dependencies(tf_cc tf_cc_framework tf_cc_ops)
+
+set (pywrap_tensorflow_lib "${CMAKE_CURRENT_BINARY_DIR}/${CMAKE_BUILD_TYPE}/pywrap_tensorflow_internal.lib")
```

Comment:
In that case, let's leave it out for now, but add a `TODO` comment mentioning that we should add it later to support third-party extensions.